ui/package.json

```diff
@@ -19,6 +19,7 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/icons-material": "^5.15.15",
+    "@mui/lab": "^5.0.0-alpha.170",
```

Comment:
Is there a non-alpha version? Maybe something more stable?

Reply:
Nope looks like it's all alhpa, and a canary.
https://www.npmjs.com/package/@mui/lab?activeTab=versions